app/styles/components/_tree-list.scss

```diff
@@ -11,16 +11,21 @@
       margin-left: 2em;
 
       li {
-        font-weight: normal;
+        font-size: 14px;
```

Comment:
All font sizes are relative to $base-font-size
https://github.com/ilios/frontend/blob/edee5bb3f39931a6649daf198f663f1631a9b506/app/styles/components/_table.scss#L3